ERROR - 2023-10-06 02:06:36 --> Error connecting to the database: Unknown database 'lat_ci'
CRITICAL - 2023-10-06 02:06:36 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'lat_ci'
#0 C:\xampp2\htdocs\lat_ci\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp2\htdocs\lat_ci\system\Database\BaseBuilder.php(1539): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#2 C:\xampp2\htdocs\lat_ci\system\Model.php(509): CodeIgniter\Database\BaseBuilder->countAllResults(false)
#3 C:\xampp2\htdocs\lat_ci\system\BaseModel.php(1084): CodeIgniter\Model->countAllResults(false)
#4 C:\xampp2\htdocs\lat_ci\app\Controllers\News.php(12): CodeIgniter\BaseModel->paginate(1, 'newses')
#5 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(832): App\Controllers\News->index()
#6 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#7 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp2\htdocs\lat_ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp2\htdocs\lat_ci\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp2\\htdoc...')
#10 {main}
ERROR - 2023-10-06 02:10:34 --> Unknown column 'slug' in 'where clause'
CRITICAL - 2023-10-06 02:10:34 --> Unknown column 'slug' in 'where clause'
#0 C:\xampp2\htdocs\lat_ci\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\lat_ci\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\lat_ci\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\lat_ci\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\lat_ci\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\lat_ci\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\lat_ci\app\Controllers\News.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(832): App\Controllers\News->viewNews('codeigniter-mee...')
#8 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#9 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp2\htdocs\lat_ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp2\htdocs\lat_ci\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp2\\htdoc...')
#12 {main}
CRITICAL - 2023-10-06 02:11:20 --> Undefined index: slug
#0 C:\xampp2\htdocs\lat_ci\app\Views\news.php(20): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp2\\htdoc...', 20, Array)
#1 C:\xampp2\htdocs\lat_ci\system\View\View.php(210): include('C:\\xampp2\\htdoc...')
#2 C:\xampp2\htdocs\lat_ci\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp2\htdocs\lat_ci\system\Common.php(1089): CodeIgniter\View\View->render('news', Array, true)
#4 C:\xampp2\htdocs\lat_ci\app\Controllers\News.php(15): view('news', Array)
#5 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(832): App\Controllers\News->index()
#6 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#7 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp2\htdocs\lat_ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp2\htdocs\lat_ci\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp2\\htdoc...')
#10 {main}
CRITICAL - 2023-10-06 02:11:22 --> Undefined index: slug
#0 C:\xampp2\htdocs\lat_ci\app\Views\news.php(20): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp2\\htdoc...', 20, Array)
#1 C:\xampp2\htdocs\lat_ci\system\View\View.php(210): include('C:\\xampp2\\htdoc...')
#2 C:\xampp2\htdocs\lat_ci\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp2\htdocs\lat_ci\system\Common.php(1089): CodeIgniter\View\View->render('news', Array, true)
#4 C:\xampp2\htdocs\lat_ci\app\Controllers\News.php(15): view('news', Array)
#5 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(832): App\Controllers\News->index()
#6 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#7 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp2\htdocs\lat_ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp2\htdocs\lat_ci\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp2\\htdoc...')
#10 {main}
ERROR - 2023-10-06 02:11:26 --> Unknown column 'slug' in 'where clause'
CRITICAL - 2023-10-06 02:11:26 --> Unknown column 'slug' in 'where clause'
#0 C:\xampp2\htdocs\lat_ci\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\lat_ci\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\lat_ci\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\lat_ci\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\lat_ci\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\lat_ci\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\lat_ci\app\Controllers\News.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(832): App\Controllers\News->viewNews('codeigniter-mee...')
#8 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#9 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp2\htdocs\lat_ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp2\htdocs\lat_ci\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp2\\htdoc...')
#12 {main}
CRITICAL - 2023-10-06 02:11:33 --> Undefined index: slug
#0 C:\xampp2\htdocs\lat_ci\app\Views\news.php(20): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp2\\htdoc...', 20, Array)
#1 C:\xampp2\htdocs\lat_ci\system\View\View.php(210): include('C:\\xampp2\\htdoc...')
#2 C:\xampp2\htdocs\lat_ci\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp2\htdocs\lat_ci\system\Common.php(1089): CodeIgniter\View\View->render('news', Array, true)
#4 C:\xampp2\htdocs\lat_ci\app\Controllers\News.php(15): view('news', Array)
#5 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(832): App\Controllers\News->index()
#6 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#7 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp2\htdocs\lat_ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp2\htdocs\lat_ci\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp2\\htdoc...')
#10 {main}
CRITICAL - 2023-10-06 02:13:21 --> Class 'Codeigniter\exception\PageNotFoundException' not found
#0 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(832): App\Controllers\News->viewNews('img', 'favico.png')
#1 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#2 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp2\htdocs\lat_ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp2\htdocs\lat_ci\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp2\\htdoc...')
#5 {main}
CRITICAL - 2023-10-06 02:13:33 --> Class 'Codeigniter\exception\PageNotFoundException' not found
#0 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(832): App\Controllers\News->viewNews('Breaking News')
#1 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#2 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp2\htdocs\lat_ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp2\htdocs\lat_ci\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp2\\htdoc...')
#5 {main}
CRITICAL - 2023-10-06 02:13:40 --> Class 'Codeigniter\exception\PageNotFoundException' not found
#0 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(832): App\Controllers\News->viewNews('img', 'favico.png')
#1 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#2 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp2\htdocs\lat_ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp2\htdocs\lat_ci\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp2\\htdoc...')
#5 {main}
ERROR - 2023-10-06 02:14:18 --> Table 'lat_ci.users' doesn't exist
CRITICAL - 2023-10-06 02:14:18 --> Table 'lat_ci.users' doesn't exist
#0 C:\xampp2\htdocs\lat_ci\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\lat_ci\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\lat_ci\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\lat_ci\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\lat_ci\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\lat_ci\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\lat_ci\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(156): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\lat_ci\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(20): Myth\Auth\Authentication\LocalAuthenticator->validate(Array, true)
#8 C:\xampp2\htdocs\lat_ci\vendor\myth\auth\src\Controllers\AuthController.php(88): Myth\Auth\Authentication\LocalAuthenticator->attempt(Array, false)
#9 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(832): Myth\Auth\Controllers\AuthController->attemptLogin()
#10 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#11 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\lat_ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp2\htdocs\lat_ci\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp2\\htdoc...')
#14 {main}
ERROR - 2023-10-06 02:16:42 --> Unknown column 'users.deleted_at' in 'where clause'
CRITICAL - 2023-10-06 02:16:42 --> Unknown column 'users.deleted_at' in 'where clause'
#0 C:\xampp2\htdocs\lat_ci\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\lat_ci\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\lat_ci\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\lat_ci\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\lat_ci\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\lat_ci\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\lat_ci\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(156): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\lat_ci\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(20): Myth\Auth\Authentication\LocalAuthenticator->validate(Array, true)
#8 C:\xampp2\htdocs\lat_ci\vendor\myth\auth\src\Controllers\AuthController.php(88): Myth\Auth\Authentication\LocalAuthenticator->attempt(Array, false)
#9 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(832): Myth\Auth\Controllers\AuthController->attemptLogin()
#10 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#11 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\lat_ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp2\htdocs\lat_ci\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp2\\htdoc...')
#14 {main}
ERROR - 2023-10-06 02:19:26 --> Table 'lat_ci.auth_logins' doesn't exist
CRITICAL - 2023-10-06 02:19:26 --> Table 'lat_ci.auth_logins' doesn't exist
#0 C:\xampp2\htdocs\lat_ci\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `au...', 0)
#1 C:\xampp2\htdocs\lat_ci\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `au...')
#2 C:\xampp2\htdocs\lat_ci\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `au...')
#3 C:\xampp2\htdocs\lat_ci\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `au...', Array, false)
#4 C:\xampp2\htdocs\lat_ci\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\lat_ci\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\lat_ci\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp2\htdocs\lat_ci\vendor\myth\auth\src\Authentication\AuthenticationBase.php(219): CodeIgniter\Model->insert(Array)
#8 C:\xampp2\htdocs\lat_ci\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(26): Myth\Auth\Authentication\AuthenticationBase->recordLoginAttempt('erik', '::1', NULL, false)
#9 C:\xampp2\htdocs\lat_ci\vendor\myth\auth\src\Controllers\AuthController.php(88): Myth\Auth\Authentication\LocalAuthenticator->attempt(Array, false)
#10 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(832): Myth\Auth\Controllers\AuthController->attemptLogin()
#11 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#12 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp2\htdocs\lat_ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp2\htdocs\lat_ci\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp2\\htdoc...')
#15 {main}
CRITICAL - 2023-10-06 02:21:26 --> Class 'Codeigniter\exception\PageNotFoundException' not found
#0 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(832): App\Controllers\News->viewNews('img', 'favico.png')
#1 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#2 C:\xampp2\htdocs\lat_ci\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp2\htdocs\lat_ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp2\htdocs\lat_ci\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp2\\htdoc...')
#5 {main}
